src/abi/GyroEV2.ts

```diff
@@ -0,0 +1,1970 @@
+export const gyroEV2Abi = [
+        {
+            inputs: [
```

Comment:
Indentation here is not properly set. Maybe it's a prettier issue? Can you double check this on the abi files you added?
I know it seems a nuisance, but if I simply open and close the file on my end, I add changes to the whole file 😅

Reply:
Thanks for highlithing. I have adapted my prettier settings.

Reply:
I'm not really sure what happened. Abi files were formatted correctly, but `onChainPoolDataViaReadContract.ts` was not. I updated the file over here, but it would be nice to double check your prettier next time you make a change 😉.